index.html

```diff
@@ -605,7 +605,224 @@ <h2>Assertion Method</h2>
 
   </section>
 
+  <section class="informative">
+    <h2>Data Model and Representations</h2>
+
+    <p class="note">
+      See [[DID-CORE]] for normative requirements associated with the
+      <a href="https://www.w3.org/TR/did-core/#data-model">Data Model</a>.
+    </p>
+
+    <p class="note">
+      See [[DID-CORE]] for normative requirements associated with
+      <a href="https://www.w3.org/TR/did-core/#representations">Representations</a>.
+    </p>
+
+    <p>
+      The DID document data model contains “properties” such as `id`, `verificationMethod` or `service`, which are
+      defined in a way that is independent of a concrete representation. There are a number of known representations,
+      such as JSON and JSON-LD (defined by DID Core), and CBOR (defined by a DID WG Note). Additional representations
+      are possible and may be defined in the future (such as XML, YaML, CBOR-LD, etc.). For each of these
+      representations, a combination of production and consumption rules defines the concrete syntax of properties, but
+      the semantics of properties are independent of any given representation. In other words, in every representation
+      of a DID document, there is always a way to represent a `service` or a `verificationMethod`, and their meanings
+      are consistent across conformant representations.
+    </p>
+
+    <p>
+      In addition to “properties”, a DID document may also contain “representation-specific entries” which may be
+      required by certain representations in order to be able to fully represent the DID document data model and its
+      properties. For example, the JSON-LD representation (with media type `application/did+ld+json`) requires the
+      `@context` representation-specific entry, in order to correctly express the semantics of properties such as `id`,
+      `verificationMethod` or `service` within the JSON-LD representation. This enables JSON-LD’s semantics which are
+      based on a decentralized, permissionless, open-world data model.
+    </p>
+
+    <p>
+      In contrast to this, the JSON representation (with media type `application/did+json`) does not have this semantic
+      capability; instead, (non-LD) JSON documents always need agreement on semantics in some out-of-band way that is
+      not part of the document itself. In the case of DID documents using the JSON representation, those semantics are
+      established by the <a href="https://w3c.github.io/did-spec-registries/">DID Spec Registries</a>. Therefore,
+      establishing semantics of DID document properties in the JSON representation does not require any
+      “representation-specific entries”, as is the case with the JSON-LD representation. The same is true for the CBOR
+      representation.
```

Comment:
```suggestion
      In contrast, the (non-LD) JSON representation (with media type `application/did+json`) does not require the
      `@context` entry, and when that entry is not present, does not contain nor communicate the semantics of other
      entries; instead, (non-LD) JSON documents always need agreement on semantics in some out-of-band way that is
      not part of the document itself. In DID documents using the JSON (non-LD) representation, those semantics are
      often established by the <a href="https://w3c.github.io/did-spec-registries/">DID Spec Registries</a>.
      Establishing semantics of DID document properties in the (non-LD) JSON DID representation does not require
      the representation-specific `@context` entry, as is required by the JSON-LD representation.
    </p>

    <p>
      Similar holds true for the CBOR and CBOR-LD representations.
```

Reply:
Thanks, I applied most of this suggested change in https://github.com/w3c/did-imp-guide/pull/8/commits/0cd2e4059884b57dd046f4cf078fe6855f82f68f.